samples/cellular/smp_svr/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
It would be good if these 2 projects were linked so that when you build one, it would build the other, will have to have a look at doing this post-sysbuild integration

Reply:
Ah OK sounds interesting. Need to investigate how to do this.

Reply:
Not something for now, way into the future